jsk_robot_common/jsk_robot_startup/package.xml

```diff
@@ -22,8 +22,10 @@
   <run_depend>actionlib_msgs</run_depend>
   <run_depend>app_manager</run_depend>
   <run_depend>dynamic_reconfigure</run_depend>
+  <run_depend>gdrive_ros</run_depend>
```

Comment:
do we need this?

Reply:
Sorry, it is already written in `google_chat_ros`